modules/react/layout/index.ts

```diff
@@ -3,19 +3,23 @@ import {DeprecatedColumn} from './lib/Column';
 
 export { DeprecatedLayout, DeprecatedColumn};
 export * from './lib/Box';
+export * from './lib/utils/background';
 export * from './lib/utils/border';
 export * from './lib/utils/color';
-export {DepthStyleProps} from './lib/utils/depth';
+export {DepthStyleProps, depthStyleFnConfigs} from './lib/utils/depth';
+export * from './lib/utils/flex';
 export * from './lib/utils/flexItem';
-export * from './lib/utils/font';
+export * from './lib/utils/grid';
+export * from './lib/utils/gridItem';
 export * from './lib/utils/layout';
+export * from './lib/utils/other';
 export * from './lib/utils/position';
-export {SpaceStyleProps} from './lib/utils/space';
+export {SpaceStyleProps, spaceStyleFnConfigs} from './lib/utils/space';
 export * from './lib/utils/text';
 export * from './lib/Flex';
 export * from './lib/Grid';
 export * from './lib/Stack';
 export type { FlexStyleProps } from './lib/utils/flex';
 export type { GridStyleProps } from './lib/utils/grid';
-export type { GridItemStyleProps } from './lib/utils/gridItem';
+export type { GridItemStyleProps, gridItemStyleFnConfigs } from './lib/utils/gridItem';
```

Comment:
We're exporting these style function configs so our `StylePropTable` can use them to build prop tables.